packages/demo/src/components/examples/DatePickerExamples.tsx

```diff
@@ -31,19 +31,19 @@ DatePickerExamples.title = 'DatePicker';
 // start-print
 
 const CalendarComponent: React.FunctionComponent = () => (
-    <Section title="Calendar component">
+    <Section level={1} title="Calendar component">
```

Comment:
How come you're adding the level when 1 is the default? Is it just to make it more obvious since its the code block?

Reply:
Habits :sweat_smile: Other sections had the level so xD 
I see much more clean up to do, but yeah I didn't want to go to overboard since we are going to remake the datePicker soon :eyes: